### PR TITLE
test(permissions/mod): Add invalid case

### PR DESCRIPTION
**Pull request by wafuwafu13**
- ~~Changed filename `test.ts` -> `mod_test.ts`~~
- Add invalid case

### CODE REVIEW

File: permissions/mod_test.ts [hunk 1]
```diff
@@ -42,3 +42,32 @@ Deno.test({
...
+Deno.test({
+  name: "grantOrThrow invalid permissionDescriptor name",
+  fn() {
+    assertThrowsAsync(
+      () => {
+        // deno-lint-ignore no-explicit-any
+        return grantOrThrow({ name: "nett" } as any);
+      },
+      TypeError,
+      'The provided value "nett" is not a valid permission name',
```
Comment (wafuwafu13) [inline permissions/mod_test.ts:68]: [memo]
https://github.com/denoland/deno/blob/b095157c1d9ffd979a82f559aa488e35c2e7f392/runtime/js/40_permissions.js#L195-L201

**Comment by caspervonb** [comment]
Why rename?

**Comment by wafuwafu13** [comment]
I was wondering if the names of the logic files corresponded to the names of the test files.
ex) 'fs/copy.ts' <=> `fs/copy_test.ts`
But in the case of `mod.ts`, I didn't have to do that, so I put it back. https://github.com/denoland/deno_std/pull/1314/commits/4a79746c807f22359cd8ec4b08555396b3e889d5

File: permissions/test.ts [hunk 2]
```diff
@@ -42,3 +42,32 @@ Deno.test({
     await grantOrThrow([{ name: "net" }, { name: "env" }]);
   },
 });
+
+Deno.test({
+  name: "grantOrThrow invalid argument",
+  fn() {
+    assertThrowsAsync(
```
Comment (kt3k) [inline permissions/test.ts:49]: nit: assertThrowsAsync is now deprecated. Please use assertRejects instead

**Review by kt3k — COMMENTED** [review]
@wafuwafu13 Thanks for your contribution. Left one comment

**Review by kt3k — APPROVED** [review]
@wafuwafu13 LGTM. Thanks!

**Pull request merged**